evm_arithmetization/src/cpu/kernel/asm/mpt/hash/hash.asm

```diff
@@ -117,21 +118,24 @@ encode_node:
     PANIC
 
 global encode_node_empty:
-    // stack: node_type, node_payload_ptr, encode_value, cur_len, retdest
-    %pop3
+    // stack: node_type, node_payload_ptr, rlp_start, encode_value, cur_len, retdest
+    %pop4
     %stack (cur_len, retdest) -> (retdest, @ENCODED_EMPTY_NODE_ADDR, 1, cur_len)
```

Comment:
mixing the pop4 with the stack macro may possibly save cycles?

Reply:
I tried it, no difference. I'll keep it as is for a shorter macro.